proc_scripts/phasing.py

```diff
@@ -2,7 +2,6 @@
 from pyspecdata import *
 from matplotlib.patches import Ellipse
 from scipy.optimize import minimize
-fl = figlist_var()
```

Comment:
this line shouldn't be present in anything but a third-level script.  fl should always be passed as an argument to any second/first level functions.  Otherwise, these are creating a **new** figure list instance, which is very confusing.